adapter_pipelines/submit.wdl

```diff
@@ -64,6 +64,7 @@ task create_submission {
   Boolean record_http
   String pipeline_tools_version
   Boolean add_md5s
+  String runtime_environment
```

Comment:
Reminder: we might need to change the submit_stub as well so the test can pass

Reply:
This PR does not add any new parameters to the submit subworkflow, so we do not need to change the submit_stub wdl

Reply:
Oh, i must be looking at the wrong variable name!